src/components/MasterItems.js

```diff
@@ -5,6 +5,19 @@ import _ from 'underscore';
 
 const MasterItemsSelect = createFactory(MasterItemsSelectComponent);
 
+function entriesByQuantity(entries) {
```

Comment:
Agree with handling client side, but the function was a little confusing at first. What do you think about simplifying a bit to something like

```
const duplicate = entry => R.repeat(entry, entry.quantity || 1);
return R.chain(duplicate, entries);
```

Or if we need to delete the `quantity` prop, change `entry` to `R.dissoc('quantity', entry)` as first argument to `R.repeat`.